### DISCUSSION TITLE
I am unable to install Kepler. It seems some issue with Kepler's image. Getting status "ContainerCreating"

### DISCUSSION THREAD

**Discussion by aniketkadukar2** [sustainable-computing-io/kepler#1312] [General]
I am unable to install Kepler. It seems some issue with Kepler's image. Getting status "ContainerCreating"

**Comment by rootfs** [comment]
@aniketkadukar2 can you describe how you got started? did you go through steps https://sustainable-computing.io/?

**Comment by harisubash** [comment]
@aniketkadukar2 

I hit a similar issue where the pod wasn't coming up because the firewall was blocking quay.io and as a result, pulling the container image was failing. Once we added an exception to that, I was able to deploy it successfully.
